web/src/styles/app/color.css

```diff
@@ -7,22 +7,22 @@
   background-color: #e1f3f8;
 }
 
-.bg-light-blue {
+.bg-blue-light {
```

Comment:
better/easier for interpolation (`${color}-light`)

Reply:
👍 One question, it looks like the background color for the individual activities was lost.  Does it need to be updated to reverse the color/modifier in the `className` too?

Reply:
fixed